### PR TITLE
Pet tool gauge chart

### PR DESCRIPTION
**Pull request by mila-proenviro**
1. new chart added to pet tool log in - gauge chart
2. initChartGauge() added to filter the data
3. charts positioned in a grid and if() statement
4. responsive chart size added
5. reposnsive col size added

### CODE REVIEW

File: src/app/pages/envirotrack/report/envirotrack-report-pie/envirotrack-report-pie.component.ts [hunk 1]
```diff
@@ -261,6 +261,7 @@ export class EnvirotrackReportPieComponent implements OnInit {
       if(this.dateFilter){
         this.filteredData = this.data.filter((x:any) => moment(x.date).isBetween(moment(this.months[this.months.length-1]).subtract(this.dateFilter,'months'), moment(this.months[this.months.length+1])))
         this.chartX = this.months.filter((x:any) => moment(x).isBetween(moment(this.months[this.months.length-1]).subtract(this.dateFilter,'months'), moment(this.months[this.months.length+1])))
+        this.chartX = this.months.filter((x:any) => moment(x).isBetween(moment(this.months[this.months.length-1]).subtract(this.dateFilter,'months'), moment(this.months[this.months.length+1])))
```
Comment (adamshelley31) [inline src/app/pages/envirotrack/report/envirotrack-report-pie/envirotrack-report-pie.component.ts:264]: This line is duplicated?

Comment (adamshelley31) [reply src/app/pages/envirotrack/report/envirotrack-report-pie/envirotrack-report-pie.component.ts:264]: @mila-proenviro

**Review by adamshelley31 — CHANGES_REQUESTED** [review]
Check duplicate line in envirotrack-report-pie.component.ts

File: src/app/pages/envirotrack/report/envirotrack-report-pie/envirotrack-report-pie.component.ts (hunk 1 above)
Comment (mila-proenviro) [reply src/app/pages/envirotrack/report/envirotrack-report-pie/envirotrack-report-pie.component.ts]: done

**Pull request merged**